price-feeder/oracle/provider/crypto.go

```diff
@@ -317,8 +319,10 @@ func (p *CryptoProvider) pong(heartbeatResp CryptoHeartbeatResponse) {
 		Method: cryptoHeartbeatReqMethod,
 	}
 
-	if err := p.wsc.SendJSON(heartbeatReq); err != nil {
-		p.logger.Err(err).Msg("could not send pong message back")
+	for _, conn := range p.wsc.connections {
```

Comment:
This could potentially be an issue as well since the heartbeat request has to match the response ID.